autogen/agentchat/contrib/critic_agent.py

```diff
@@ -0,0 +1,42 @@
+from typing import Callable, Dict, Literal, Optional, Union
+
+from autogen.agentchat.assistant_agent import AssistantAgent
+from autogen.runtime_logging import logging_enabled, log_new_agent
+
+
+class CriticAgent(AssistantAgent):
```

Comment:
Should we inherit from `ConversableAgent` instead of the `AssistantAgent`? The implementation would almost be the same.

Reply:
@BeibinLi  what would the benefits of that?

Reply:
AssistantAgent is a short-cut for not writing system messages and other parameters, but it can subject to changes in the future. You probably don't want your subclass to get unexpected behavior changes.

Reply:
That makes sense, I'll make the change in the next iteration.